Update documentation regarding no longer processing duplicate webhooks. (#271)
hashicorp/terraform-docs-common: website/docs/cloud-docs/vcs/index.mdx

## Viewing events

> VCS events describe changes within your organization for VCS-related actions, such as when a webhook is received with a previously processed commit SHA. Displayed events are limited to the past 30 days.

Reviewer: ```suggestion
VCS events describe changes within your organization for VCS-related actions, such as when Terraform Cloud receives a webhook with a previously processed commit SHA. The VCS events page only displays events from the past 30 days.
```
